[DRAFT] Add Vilnius 2019 schedule

mxsasha opened the pull request:
The day does end quite early (schedule page still says 18:00, we need to update this if this is the final schedule).
The original planned time was from 10-18, but we have 3 hours of talks, 40 minutes of lightning talks, an hour of lunch, an hour of snack breaks, ~30 minutes of speaker switching. So that's just over 6 hours - we just can't really fill 8 hours, hence the early ending.

Also, please double check that I didn't make any timing calculation errors.
(snack breaks are 30 minutes on purpose for a bit more padding)

mxsasha committed:
Add Vilnius 2019 schedule

mxsasha committed:
Activate "see schedule" button for Vilnius 2019

mxsasha committed:
Fix speaker links in Vilnius 2019 schedule

thatdocslady commented:
Thanks @mxsasha looks good!

I'm wondering if extending the snack breaks might not be too much for an already-stretched schedule. Maybe instead, we can shrink them back and add 1-2 hours of post-conference networking time (let's say 16:00-18:00) in lieu of the welcome reception that we're not formally hosting. 

Thoughts?

mxsasha commented:
Yes, that sounds good. We can also cut lightning talks to 30 minutes (now 40). That saves us a total of half an hour, ending at 16:10, and then networking time until 18:00? Or 17:30.
(needs a better name for the schedule probably)

thatdocslady commented:
@mxsasha great! Let's do that :) I say ending at 17:30 is fine, but maybe 18:00 looks better on the schedule? As for the name, how about something like "Post-conference social time" or "Informal networking"... I'll think about it some more :)

mxsasha committed:
Update Vilnius 2019 schedule

mxsasha commented:
I added it for now as informal networking until 18:00. Someone should double-check I updated all the start times correctly, snacks are 20 minutes now, lightning talks 30.

aidiss commented:
I agree regarding post-conference social time.
Do you need anything from me?

aidiss reviewed (approved):
Looks good

thatdocslady commented:
Thanks @aidiss, I'll merge this PR now and start working on the blog post.
